bikeshed/test.py

```diff
@@ -95,7 +101,7 @@ def equalOrEmpty(a, b):
 
 def rebase(files=None):
     if not files:
-        files = glob.glob(config.scriptPath() + "/../tests/*.bs")
+        files = findTestFiles()
     for path in files:
         _,_,name = path.rpartition("/")
         p("Rebasing {0}".format(name))
```

Comment:
There's a problem here and with `p(testName)` above, in that just the filename is printed but the path relative to tests/ would be better. But how, given that the paths might also be outside? Use `os.path.abspath` on all paths and look for a prefix?

Reply:
Yeah, removing a common prefix is pretty easy.